website/templates/profile/account.mako

```diff
@@ -95,8 +95,8 @@
                                             This location will be applied to new projects and components. It will not affect existing projects and components.
                                         </p>
                                     </div>
+                                    <input type="submit" class="btn btn-primary" data-bind="click: $root.updateDefaultStorageLocation.bind($root)" value="Update location" />
```

Comment:
What was this button moved?

Also, you should use the "submit" data binding (see the link in my previous comment).

Reply:
There's no link in your earlier comment.

Reply:
Woops, meant to link to https://knockoutjs.com/documentation/submit-binding.html .

The button was already inside the `<form>` tag. If I remember Bootstrap correctly, the submit button isn't supposed to go in a `div.form-group`. https://getbootstrap.com/docs/3.3/css/#forms